src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,43 @@
+package duke;
```

Comment:
Suggesting the separation of your classes into multiple packages instead of a sole Duke package that has multiple classes- allows for better organization of code

Reply:
I will take your suggestion into consideration